CRM/Report/Form.php

```diff
@@ -596,20 +595,8 @@ public function preProcessCommon() {
     if ($this->_id) {
       $this->assign('instanceId', $this->_id);
       $params = ['id' => $this->_id];
-      $this->_instanceValues = [];
-      CRM_Core_DAO::commonRetrieve('CRM_Report_DAO_ReportInstance',
-        $params,
-        $this->_instanceValues
-      );
-      if (empty($this->_instanceValues)) {
-        CRM_Core_Error::fatal("Report could not be loaded.");
-      }
-      $this->_title = $this->_instanceValues['title'];
```

Comment:
@mlutfy this line that sets the title is not in the isInstancePermissioned function

Otherwise it all seems good

Reply:
Oh I see - you mention it specifically - will double check it

Reply:
OK - it's accessed from getTitle - although that is called less often than it should be